src/dialog.js

```diff
@@ -147,6 +147,7 @@ genotet.dialog.createNetwork_ = function() {
       var params = {
         type: genotet.data.ListQueryType.NETWORK
       };
+      params = {data: JSON.stringify(params)};
```

Comment:
This seems to be heavily repeated code. Consider refactoring this to a wrapper function, or at least mark a TODO for the moment.